x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_settings.tsx

```diff
@@ -21,56 +19,25 @@ export function LayerSettings({
   activeVisualization: Visualization;
   layerConfigProps: VisualizationLayerWidgetProps;
 }) {
-  const [isOpen, setIsOpen] = useState(false);
+  const description = activeVisualization.getDescription(layerConfigProps.state);
 
-  if (!activeVisualization.renderLayerContextMenu) {
-    return null;
-  }
-
-  const a11yText = (chartType?: string) => {
-    if (chartType) {
-      return i18n.translate('xpack.lens.editLayerSettingsChartType', {
-        defaultMessage: 'Edit layer settings, {chartType}',
-        values: {
-          chartType,
-        },
-      });
+  if (!activeVisualization.renderLayerHeader) {
+    if (!description) {
+      return null;
     }
-    return i18n.translate('xpack.lens.editLayerSettings', {
-      defaultMessage: 'Edit layer settings',
-    });
-  };
+    return (
+      <>
+        {description.icon && (
+          <EuiIcon type={description.icon} className="lnsLayerPanel__settingsStaticHeaderIcon" />
+        )}
+        <EuiTitle size="xxs" className="lnsLayerPanel__settingsStaticHeader">
+          <h5>{description.label}</h5>
+        </EuiTitle>
+      </>
```

Comment:
I believe this block was previously using an `EuiFlexGroup`, with the icon and title in their own individual `EuiFlexItem`. I think it would be good to go back to that structure, as it makes the space more bulletproof against line breaks. Also, it would correct the current issue that appears to be happening with static headers, where the icon and title are not aligned horizontally.

![image](https://user-images.githubusercontent.com/3884767/129098818-33043fd7-26ba-45c7-91fb-5ae5cd26b113.png)